internal/config/config.go

```diff
@@ -380,6 +389,14 @@ func (t *Tunnel) SetDefaults() {
 	}
 }
 
+// SetDefaults updates npm default values
+func (n *Npm) SetDefaults() {
+	if n.Registry != "" {
+		log.Warn().Msg("npm.registry has been deprecated, please use npm.registries instead")
+		n.Registries = append(n.Registries, Registry{URL: n.Registry})
```

Comment:
But if the user does that now as indicated here, it'll only work for them with some framework version x.y.z. and above. How would they know that you're leading them into a trap ;)

Reply:
I added a function to check that. https://github.com/saucelabs/saucectl/pull/809/files#diff-54c7c1af5fa8d5db4dc49f0e8e80e93ba2b1183ba4d5c9e2e5729e6deae6a3cdR393

I'm not 100% convinced on the way of doing it, but that's probably one of the softest approach from a customer point of view.

Reply:
Yeah... I can't seem to think of a "better" approach. There's other alternatives, like bumping the config spec. That's usually a humongous effort with code refactor + doc changes.

This seems good enough.